plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -1691,7 +1691,7 @@ read_only::get_producers_result read_only::get_producers( const read_only::get_p
    for (auto p : db.active_producers().producers) {
       fc::variant row = fc::mutable_variant_object()
          ("owner", p.producer_name)
-         ("producer_key", p.block_signing_key)
```

Comment:
opted to maintain legacy results when I can by detecting a canonical 1-of-1 wtmsig and injecting the old `producer_key` field.  Noted in the comments as well so that it can be labeled as "deprecated" or something similar in the API docs